devicon.json

```
@@ -7309,6 +7309,28 @@
         "color": "#903a36",
         "aliases": []
     },
+    {
+        "name": "envoyproxy",
+        "altnames": [
+            "envoy"
+        ],
+        "tags": [
+            "middle proxy",
```

Never heard of middle proxy before. Did you mean `proxy`?
```suggestion
            "proxy",
```